src/form.js

```diff
@@ -1,9 +1,11 @@
 /* eslint-env browser */
 import { TAG as SUGGESTIONS_TAG } from "./suggestions";
+import { TAG as STATUS_TAG } from "./status";
 import { serializeForm } from "uitil/dom/forms";
 import { dispatchEvent } from "uitil/dom/events";
 import bindMethods from "uitil/method_context";
 import debounce from "uitil/debounce";
+import { insertAfter } from "./util";
```

Comment:
local > third-party modules

(the diff looks horrible for some reason: I just moved the `./util` import beneath `./status`)

```suggestion
import { insertAfter } from "./util";
import { serializeForm } from "uitil/dom/forms";
import { dispatchEvent } from "uitil/dom/events";
import bindMethods from "uitil/method_context";
import debounce from "uitil/debounce";
```

Reply:
updated